lineman/app/components/edit_group_form/edit_group_form.haml

```diff
@@ -3,76 +3,139 @@
-        %option{value: 'public_or_private', ng-if: 'validDiscussionOption("public_or_private")', translate: 'edit_group_form.privacy.public_or_private_discussions' }
-        %option{value: 'private_only', ng-if: 'validDiscussionOption("private_only")', translate: 'edit_group_form.privacy.private_discussions'}
-
-      %h2.lmo-h2{translate: 'edit_group_form.permissions.label'}
-      %strong{translate: 'edit_group_form.permissions.helptext'}
-
-      .checkbox
-        %label{for: 'group-members-can-add-members'}
-          %input.edit-group-form__members-can-add-members#group-members-can-add-members{type: 'checkbox', ng-model: 'group.membersCanAddMembers'}
-          %span{translate: 'edit_group_form.permissions.members_can_add_members'}
-
-      .checkbox
-        %label{for: 'group-members-can-create-subgroups'}
-          %input.edit-group-form__members-can-create-subgroups#group-members-can-create-subgroups{type: 'checkbox', ng-model: 'group.membersCanCreateSubgroups'}
-          %span{translate: 'edit_group_form.permissions.members_can_create_subgroups'}
-
-      .checkbox
-        %label{for: 'group-members-can-start-discussions'}
-          %input.edit-group-form__members-can-start-discussions#group-members-can-start-discussions{type: 'checkbox', ng-model: 'group.membersCanStartDiscussions'}
-          %span{translate: 'edit_group_form.permissions.members_can_start_discussions'}
-
-      .checkbox
-        %label{for: 'group-members-can-edit-discussions'}
-          %input.edit-group-form__members-can-edit-discussions#group-members-can-edit-discussions{type: 'checkbox', ng-model: 'group.membersCanEditDiscussions'}
-          %span{translate: 'edit_group_form.permissions.members_can_edit_discussions'}
-
-      .checkbox
-        %label{for: 'group-members-can-edit-comments'}
-          %input.edit-group-form__members-can-edit-comments#group-members-can-edit-comments{type: 'checkbox', ng-model: 'group.membersCanEditComments'}
-          %span{translate: 'edit_group_form.permissions.members_can_edit_comments'}
-
-      .checkbox
-        %label{for: 'group-members-can-raise-motions'}
-          %input.edit-group-form__members-can-raise-motions#group-members-can-raise-motions{type: 'checkbox', ng-model: 'group.membersCanRaiseProposals'}
-          %span{translate: 'edit_group_form.permissions.members_can_raise_motions'}
-
-      .checkbox
-        %label{for: 'group-members-can-vote'}
-          %input.edit-group-form__members-can-vote#group-members-can-vote{type: 'checkbox', ng-model: 'group.membersCanVote'}
-          %span{translate: 'edit_group_form.permissions.members_can_vote'}
+      .edit-group-form__basic
+        .edit-group-form__name
+          %label{for: 'group-name', translate: 'edit_group_form.name'}
+          %input.form-control#group-name{ng-required: true, ng-model: 'group.name', maxlength: '255'}
```

Comment:
There are some classes that should be here to enable cmd-enter submit; I think it's `lmo-primary-form-input`? For the textarea too; I'd just copy the classes that were there before.